src/main/java/jenkins/plugins/gerrit/client/GerritClientBuilder.java

```diff
@@ -0,0 +1,177 @@
+package jenkins.plugins.gerrit.client;
+
+import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import com.jcraft.jsch.JSch;
```

Comment:
See this[1] it seems like jsch is dead

[1] https://www.eclipse.org/lists/egit-dev/msg04418.html

Reply:
Huh. Well, I'll look into apache-sshd then. Like I said, I just picked whatever was already available on the Jenkins classpath.